app/models/piece.rb

```diff
@@ -17,6 +17,63 @@ def obstructed?(destination_row, destination_column)
     fail 'Input is invalid!'
   end
 
+  def move_to!(destination_row, destination_column)
+      # byebug
+  # practice line begin here
+    
+    if occupied_space?(destination_row, destination_column) && capturable?(destination_row, destination_column)
+      game.pieces.where(row: :destination_row, column: :destination_column).update_attributes(row: nil, column: nil, in_game: false) #not sure whats proper...just woke up hour ago   :)
+     #^^^^^ this is opponent pieces ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+     
+      update_attributes(row: destination_row, column: destination_column) #move our pieces to that new location after removing opponent
+      return true
+    elsif !(occupied_space?(destination_row, destination_column))
+      update_attributes(row: destination_row, column: destination_column)
+      return true
+    else 
+      return false
+    end
+  end
+  
+  
+
+
+
+#practice line end here
+
+
+
+
+
+
+#    if occupied_space?(destination_row, destination_column)
+#      opponent = game.pieces.where(row: destination_row, column: destination_column)
+
+#      if capturable?(destination_row, destination_column)
+#        opponent.update_attributes(row: nil, column: nil, in_game: false)
+#        update_attributes(row: destination_row, column: destination_column)
+#      else
+#        return false
+#      end
+
+#    else
+
+#      update_attributes(row: destination_row, column: destination_column)
+
+#    end
+#  end
+
+  def occupied_space?(d_row, d_column)
+    return true if game.pieces.where(row: d_row, column: d_column).exists?
+    false
```

Comment:
could shorten this method up to be a single line.

`game.pieces.where(row: d_row, column: d_column).exists?`

Since that line of code returns `true` or `false`